doc/introduction/introducing_zephyr.rst

```diff
@@ -32,53 +32,94 @@ Distinguishing Features
 The Zephyr kernel offers a number of features that distinguish it from other
 small-footprint OSes:
 
-**Single address-space**
-   Combines application-specific code
-   with a custom kernel to create a monolithic image that gets loaded
-   and executed on a system's hardware. Both the application code and
-   kernel code execute in a single shared address space.
-
 **Highly configurable / Modular for flexibility**
-   Allows an application to incorporate *only*
-   the capabilities it needs as it needs them, and to specify their
-   quantity and size.
+   Allows an application to incorporate *only* the capabilities it needs as it
+   needs them, and to specify their quantity and size.
 
 **Cross Architecture**
    Supports a wide variety of :ref:`supported boards<boards>` with different CPU
    architectures and developer tools. Contributions have added support
    for an increasing number of SoCs, platforms, and drivers.
 
 **Compile-time resource definition**
-   Allows system resources
-   to be defined at compile-time, which reduces code size and
-   increases performance for resource-limited systems.
+   Allows system resources to be defined at compile-time, which reduces code
+   size and increases performance for resource-limited systems.
+
+**Optimized Device Driver Model**
+   Provides a consistent device model for configuring the drivers that are part
+   of the platform/system and a consistent model for initializing all the
+   drivers configured into the system and Allows the reuse of drivers across
+   platforms that have common devices/IP blocks
 
-**Minimal and Configurable error checking**
-   Provides minimal runtime error checking
-   to reduce code size and increase performance. An optional error-checking
-   infrastructure is provided to assist in debugging during application
-   development.
+**Device Tree Support**
+   Use of Device Tree (DTS) to describe hardware and configuration information for
+   boards. The DTS information will be used only during compile time.
+   Information about the system is extracted from the compiled DTS and used to
+   create the application image.
 
 **Memory Protection**
    Implements configurable architecture-specific stack-overflow protection,
    kernel object and device driver permission tracking, and thread isolation
    with thread-level memory protection on x86, ARC, and ARM architectures,
    userspace, and memory domains.
 
+   For platforms without MMU/MPU and memory constrained devices support for
```

Comment:
Better would be:

    For platforms without MMU/MPU and memory constrained devices, supports
    combining ...